sdr/liquid_dsp/Makefile

```diff
@@ -0,0 +1,247 @@
+CSRCS += $(LIQUID_DSP_UNPACK)/src/modem/src/gmskmod.c
+CSRCS += $(LIQUID_DSP_UNPACK)/src/modem/src/modemcf.c
+CSRCS += $(LIQUID_DSP_UNPACK)/src/modem/src/modem_utilities.c
+CSRCS += $(LIQUID_DSP_UNPACK)/src/modem/src/modem_apsk_const.c
+CSRCS += $(LIQUID_DSP_UNPACK)/src/modem/src/modem_arb_const.c
+
+CSRCS += $(LIQUID_DSP_UNPACK)/src/multichannel/src/firpfbch_crcf.c
+CSRCS += $(LIQUID_DSP_UNPACK)/src/multichannel/src/firpfbch_cccf.c
+CSRCS += $(LIQUID_DSP_UNPACK)/src/multichannel/src/ofdmframe.common.c
+CSRCS += $(LIQUID_DSP_UNPACK)/src/multichannel/src/ofdmframegen.c
+CSRCS += $(LIQUID_DSP_UNPACK)/src/multichannel/src/ofdmframesync.c
+
+CSRCS += $(LIQUID_DSP_UNPACK)/src/nco/src/nco_crcf.c
+CSRCS += $(LIQUID_DSP_UNPACK)/src/nco/src/nco.utilities.c
+CSRCS += $(LIQUID_DSP_UNPACK)/src/nco/src/synth_crcf.c
+
+CSRCS += $(LIQUID_DSP_UNPACK)/src/optim/src/chromosome.c
+CSRCS += $(LIQUID_DSP_UNPACK)/src/optim/src/gasearch.c
+CSRCS += $(LIQUID_DSP_UNPACK)/src/optim/src/gradsearch.c
+CSRCS += $(LIQUID_DSP_UNPACK)/src/optim/src/optim.common.c
+CSRCS += $(LIQUID_DSP_UNPACK)/src/optim/src/qnsearch.c
+CSRCS += $(LIQUID_DSP_UNPACK)/src/optim/src/utilities.c
+
+CSRCS += $(LIQUID_DSP_UNPACK)/src/quantization/src/compand.c
+CSRCS += $(LIQUID_DSP_UNPACK)/src/quantization/src/quantizercf.c
+CSRCS += $(LIQUID_DSP_UNPACK)/src/quantization/src/quantizerf.c
+CSRCS += $(LIQUID_DSP_UNPACK)/src/quantization/src/quantizer.inline.c
+
+CSRCS += $(LIQUID_DSP_UNPACK)/src/random/src/rand.c
+CSRCS += $(LIQUID_DSP_UNPACK)/src/random/src/randn.c
+CSRCS += $(LIQUID_DSP_UNPACK)/src/random/src/randexp.c
+CSRCS += $(LIQUID_DSP_UNPACK)/src/random/src/randweib.c
+CSRCS += $(LIQUID_DSP_UNPACK)/src/random/src/randgamma.c
+CSRCS += $(LIQUID_DSP_UNPACK)/src/random/src/randnakm.c
+CSRCS += $(LIQUID_DSP_UNPACK)/src/random/src/randricek.c
+CSRCS += $(LIQUID_DSP_UNPACK)/src/random/src/scramble.c
+
+CSRCS += $(LIQUID_DSP_UNPACK)/src/sequence/src/bsequence.c
+CSRCS += $(LIQUID_DSP_UNPACK)/src/sequence/src/msequence.c
+
+CSRCS += $(LIQUID_DSP_UNPACK)/src/utility/src/bshift_array.c
+CSRCS += $(LIQUID_DSP_UNPACK)/src/utility/src/byte_utilities.c
+CSRCS += $(LIQUID_DSP_UNPACK)/src/utility/src/msb_index.c
+CSRCS += $(LIQUID_DSP_UNPACK)/src/utility/src/pack_bytes.c
+CSRCS += $(LIQUID_DSP_UNPACK)/src/utility/src/shift_array.c
+CSRCS += $(LIQUID_DSP_UNPACK)/src/utility/src/utility.c
+
+CSRCS += $(LIQUID_DSP_UNPACK)/src/vector/src/vectorf_add.port.c
+CSRCS += $(LIQUID_DSP_UNPACK)/src/vector/src/vectorf_norm.port.c
+CSRCS += $(LIQUID_DSP_UNPACK)/src/vector/src/vectorf_mul.port.c
+CSRCS += $(LIQUID_DSP_UNPACK)/src/vector/src/vectorf_trig.port.c
+CSRCS += $(LIQUID_DSP_UNPACK)/src/vector/src/vectorcf_add.port.c
+CSRCS += $(LIQUID_DSP_UNPACK)/src/vector/src/vectorcf_norm.port.c
+CSRCS += $(LIQUID_DSP_UNPACK)/src/vector/src/vectorcf_mul.port.c
+CSRCS += $(LIQUID_DSP_UNPACK)/src/vector/src/vectorcf_trig.port.c
+
+CSRCS += $(LIQUID_DSP_UNPACK)/src/libliquid.c
+
+context:: $(LIQUID_DSP_UNPACK)/.patch
```

Comment:
let's skip the download if .git exist

Reply:
Not address yet.

Reply:
Do you have a suggestion od solution? Thanks.

Reply:
like this:
https://github.com/apache/nuttx-apps/pull/1625/files#diff-e875ca585515821e39295cd459a0513f16b3d57fc91b5d2afc48d66cb776d03bR143-R150
since you do the suggestion in nng, I assume you know it.